src/vma/dev/qp_mgr.h

```diff
@@ -62,6 +63,12 @@ class ring_eth_cb;
 #define MAX_SUPPORTED_IB_INLINE_SIZE	884
 #endif
 
+struct qp_mgr_desc {
+	const ring_simple* ring;
```

Comment:
Consider removing `const`. In this case, no need to convert types in qp_mgr constructor.